bundles/framework/myplacesimport/UserLayersList.jsx

```diff
@@ -90,7 +97,8 @@ export const UserLayersList = ({ data = [], controller, loading }) => {
                 key: item.getId(),
                 name: Oskari.util.sanitize(item.getName()),
                 description: Oskari.util.sanitize(item.getDescription()),
-                source: Oskari.util.sanitize(item.getSource())
+                source: Oskari.util.sanitize(item.getSource()),
+                created: Oskari.util.formatDate(item.getCreated())
```

Comment:
See comment about render function:
```suggestion
                created: item.getCreated()
```